src/setupTest.js

```diff
@@ -18,3 +18,8 @@ global.window.matchMedia = matchMediaMock.create();
 
 jest.mock('@edx/frontend-platform/logging');
 jest.mock('@edx/frontend-platform/analytics');
+
+// Upgrading to Node16 shows unhandledPromiseRejection warnings as errors so adding a handler
+process.on('unhandledRejection', (reason, p) => {
+  console.log('Unhandled Rejection at: Promise', p, 'reason:', reason.stack);
```

Comment:
Let's ignore this ESLint warning so that it doesn't keep showing on `npm run lint`